src/main/java/hudson/plugins/sloccount/SloccountProjectAction.java

```diff
@@ -75,8 +72,8 @@ public void doIndex(final StaplerRequest request, final StaplerResponse response
      * @return the last finished build or <code>null</code> if there is no
      *         such build
      */
-    public AbstractBuild<?, ?> getLastFinishedBuild() {
-        AbstractBuild<?, ?> lastBuild = project.getLastBuild();
+    public Run<?, ?> getLastFinishedBuild() {
```

Comment:
I followed one of blog post (I think it was for Gatling plugin) - there was info that AbstractBuild and AbstractProject are converted to Run and Job.